src/API/OnboardingTasks.php

```diff
@@ -762,8 +753,14 @@ public function get_status() {
 	 * @return WP_REST_Response|WP_Error
 	 */
 	public function get_tasks() {
-		$task_lists = TaskLists::get_all();
-		return rest_ensure_response( $task_lists );
+		$lists = TaskLists::get_all();
+		$json  = array_map(
+			function( $list ) {
+				return $list->get_json();
+			},
+			$lists
+		);
+		return rest_ensure_response( apply_filters( 'woocommerce_admin_onboarding_tasks', $json ) );
```

Comment:
I'm thinking of removing this filter altogether.  Since it was recently introduced, I don't think it has been externally used yet.